### PR TITLE
Portifolio setup

### PR DESCRIPTION
**Pull request by nedith**
Project objectives:
-Understand how to parse a Figma design to create a UI.
Use Flexbox to place elements on the page.
-Build a personal portfolio site.
-Use images and backgrounds to enhance the look of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+</head>
+<body>
+      <nav class="nav-bar">
+          <div class="logo">
+            <a href="#">My Logo</a>
+          </div>
+          <img src="./images/Union.png" alt="">
+         
+      </nav>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Please enclose this block in a `header` tag since the navbar will always be at the top of the page 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                      If you like what you see and have a project you need coded, 
+                      don’t hestiate to contact me. </p> 
+                </article>
+                 <h4>Lets connect</h4>
+                  <ul> 
+                      <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                       <li><i class="fa fa-linkedin" aria-hidden="true"></i></li> 
+                       <li><i class="fa fa-maxcdn" aria-hidden="true"></i></li>
+                        <li><i class="fa fa-github" aria-hidden="true"></i></li> 
+                        <li><i class="fa fa-hand-peace-o" aria-hidden="true"></i></li>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Each of these `li` tags should be enclosed in an `a` tag sicne they are links to your socials 🏄

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                </article>
+                 <h4>Lets connect</h4>
+                  <ul> 
+                      <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                       <li><i class="fa fa-linkedin" aria-hidden="true"></i></li> 
+                       <li><i class="fa fa-maxcdn" aria-hidden="true"></i></li>
+                        <li><i class="fa fa-github" aria-hidden="true"></i></li> 
+                        <li><i class="fa fa-hand-peace-o" aria-hidden="true"></i></li> 
+                    </ul> 
+                </section>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] There should be a padding around this section as seen in the Figma. 
### Your Implementation
image
### Required from the Figma Template
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @nedith  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks are passing
- Clean code 🧹 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- Please update your README to have actual information but not the default values provided by thremicroverse template

File: index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                      If you like what you see and have a project you need coded, 
+                      don’t hestiate to contact me. </p> 
+                </article>
+                 <h4>Lets connect</h4>
+                  <ul> 
+                    <li><a href="#"></a><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                    <li><a href="#"></a><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+                    <li><a href="#"></a><i class="fa fa-maxcdn" aria-hidden="true"></i></li>
+                    <li><a href="#"></a><i class="fa fa-github" aria-hidden="true"></i></li>
+                    <li><a href="#"></a><i class="fa fa-hand-peace-o" aria-hidden="true"></i></li>
```
Comment (OLIPLICHE) [inline index.html]: - [ ] Please each icon has a link `<a>` tag. Example: `<li><a><i class= ... okay-nice...></i></a></li>`

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @nedith , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+                </article>
+                 <h4>Lets connect</h4>
+                  <ul> 
+                    <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-maxcdn" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-hand-peace-o" aria-hidden="true"></i></a></li>
+                    </ul> 
+                </section>
```
Comment (koachcode) [inline index.html:42]: - [Opotional] Please use the proper icon color and looks for your project design just as it is in the Figma design  

##### Figma design 
Screenshot from 2022-05-26 21-02-37

##### Your design
Screenshot from 2022-05-26 21-03-37

- [Optional] Please add the correct spacing between sections and elements in your project design, to be similar to the Figma design. 

- [Optional] Please work on the font sizes as well to be similar to that of the Figma design

##### Figma design 
Screenshot from 2022-05-26 21-05-48

##### Your design
Screenshot from 2022-05-26 21-06-33

- [Optional] Please make your readme professional by editing it and putting in the writeup that best describes your project and do not forget to add your social media links as well

**Review by koachcode — APPROVED** [review]
Hi @nedith , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- Feel free to reach out to me on slack, if you find any feedback on this project confusing via: **Kanu mike chibundu**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**